### ISSUE TITLE
*render() generator function throws TypeError: value.bind is not a function

### ISSUE THREAD

**Issue by kvcrawford** [kriszyp/alkali#47]
![screen shot 2017-08-15 at 1 22 48 pm](https://user-images.githubusercontent.com/504158/29334758-5fb3b06e-81bd-11e7-99bd-4a09d8dfd7ac.png)

**Comment by kvcrawford** [comment]
```js
let selectionVar = new VSet([0, 5, 10])

function* render() {
  let selection = yield selectionVar
  let i = this.dataset.index

  this.classList.toggle('selected', selection.has(i))
}

for (let i=0; i < 100; i++) {
  new Button({
    dataset: {
      index: i
    },
    render: render
  })
}
```

**Comment by kriszyp** [comment]
Released in v0.16.7.
